src/ontology/obi-edit.owl

```diff
@@ -29712,7 +29712,7 @@ MHB 3-5-13: Need to review axiom on this class in light of clarification that it
     <!-- http://purl.obolibrary.org/obo/OBI_1110095 -->
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/OBI_1110095">
-        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0600014"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OBI_0600005"/>
```

Comment:
shouldn't we change the description too?

The collection of a blood specimen from some organism?